test/metabase/test/data/dataset_definitions/useful-dates.edn

```diff
@@ -0,0 +1,18 @@
+;; useful dates for driver tests
```

Comment:
I made the edn file here so we can let this grow and 3rd party drivers can use these dates as well.